html/semantics/links/following-hyperlinks/activation-behavior.window.js

```diff
@@ -0,0 +1,43 @@
+["a",
+ "area"].forEach(type => {
+
+  const followed = type === "a" ? true : false;
+  async_test(t => {
+    const target = document.createElement("iframe"),
+          link = document.createElement(type);
+    t.add_cleanup(() => target.remove());
+    target.name = "certifiedrandom" + type;
+    link.target = "certifiedrandom" + type;
+    link.href = "/";
+    document.body.appendChild(target);
+    target.onload = t.step_func(() => {
```

Comment:
This test will pass after 500 ms if followed is true and the target never loads. That seems wrong?

Reply:
Good, seems that found a bug in Fx.